bridge/client/app/_views/ktb-settings-view/ktb-integration-view/ktb-modify-uniform-subscription/ktb-modify-uniform-subscription.component.ts

```diff
@@ -79,11 +83,16 @@ export class KtbModifyUniformSubscriptionComponent implements OnDestroy, Pending
   public unsavedDialogState: null | 'unsaved' = null;
   private isFilterDirty = false;
 
+  public subscriptionDeletionData: DeleteData = {
+    type: DeleteType.SUBSCRIPTION,
+  };
```

Comment:
Because we just need one single property here, it would be sufficient to set `deleteType = DeleteType.Subscrption` or `DeleteType = DeleteType;`